ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,15 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    if self.current >= self.capacity:
+    	self.current = 0
+    self.storage[self.current] = item
+    self.current += 1
 
   def get(self):
-    pass
\ No newline at end of file
+  	result = []
+  	for i in range(0, self.capacity):
+  		item = self.storage[i]
+  		if item is not None:
+  			result.append(item)
+  	return result
```

Comment:
good job solving this. You could also do it in a one liner:

return [x for x in self.storage if x is not None]